lib/Conversion/TTNNToEmitC/TTNNToEmitC.cpp

```diff
@@ -200,6 +200,10 @@ void populateTTNNToEmitCPatterns(mlir::MLIRContext *ctx,
   patterns.add<DefaultOpConversionPattern<ttnn::SumOp>>(typeConverter, ctx);
   patterns.add<DefaultOpConversionPattern<ttnn::MeanOp>>(typeConverter, ctx);
 
+  // Conv ops
+  //
+  patterns.add<DefaultOpConversionPattern<ttnn::Conv2dOp>>(typeConverter, ctx);
```

Comment:
No maxpool conversion?

Reply:
Not yet, I'll bringup the maxpool op when tt-metal gets uplifted. Should I delete the op definitions from this branch?

Reply:
Yeah I think so, we want to get in the habit of having a unit test with every commit, with some exceptions.  But this is a case since the max pool isn't functional anyway, it'd be better to just wait until we can test it.

Reply:
Removed maxpool dead code